src/main/java/org/takes/http/Options.java

```diff
@@ -45,8 +48,19 @@
  * <p>The class is immutable and thread-safe.
  *
  * @author Yegor Bugayenko (yegor@teamed.io)
- * @version $Id$
+ * @version $Id: 27bab18e3251f714da54d31f464288973f6a101f $
  * @since 0.2
+ *
+ * @todo  #517 This class need refactoring
```

Comment:
@sebing you are trying to do too much in this PR, just revert that changes and put exclude